dts/bindings/adc/gd,gd32-adc.yaml

```diff
@@ -24,31 +24,6 @@ properties:
         RCU(Reset Control Unit) HAL provide a enum called rcu_adc_clock_enum to
         select the ADC clock source.
```

Comment:
nit: add comment that valid values are in `dts-bindings/adc/gd32xxx.h` headers.

Reply:
done.